modules/ROOT/pages/installation/windows.adoc

```diff
@@ -227,3 +227,33 @@ Invoke-Neo4jAdmin
 === Common PowerShell parameters
 
 The module commands support the common PowerShell parameter of `Verbose`.
+
+== Uninstall Neo4j
+
+In case you want to specifically remove Neo4j Desktop, refer to https://neo4j.com/docs/desktop-manual/current/troubleshooting/re-install/[Neo4j Desktop -> Re-installation of Desktop].
+
+Here are the steps to uninstall Neo4j on Windows:
+
+. (Optional) Create a xref:/backup-restore/index.adoc[backup] to avoid losing your data.
+. Stop all Neo4j processes by using the Task Manager.
+. If you have Neo4j Desktop installed, uninstall it via _Control Panel -> Programs_.
+. Make sure the folder _\Program Files\Neo4j Desktop_ no longer exists.
+. Delete all files in _\%USERPROFILE%.Neo4jDesktop_.
+. In a command window, write:
+[source, shell]
+---
+powershell
+(get-host).version
+exit
+---
++
+[NOTE]
+====
+Make sure that your system PATH includes `\windows\system32\wbem :` and that you have Powershell installed.
+====
+. In a command window, write: 
+[source, shell]
+---
+echo %PATH%
+---
```

Comment:
I don't know what this is supposed to achieve. I think the instructions you found might be wrong.